### ISSUE TITLE
Problem running Tensorflow graph using C++ API

### ISSUE THREAD

**Issue by umairalipathan** [tensorflow/tensorflow#9508]
I have retrained Inception-v3 model using (Tensorflow) Python API, and saved a standalone Graph in .pb form. I have also used a dropout layer before the final layer. I can successfully run inference on the graph in python. The code to generate predictions in python is as follows:
`softmax_tensor = sess.graph.get_tensor_by_name('final_layer/final_result/Softmax:0')
predictions = sess.run(softmax_tensor, { 'DecodeJpeg/contents:0': image_data, 'final_layer/dropout/Placeholder:0': 1.})`
The C++ counterpart of the python code is as follows:
` string input_layer = "Mul";
  string output_layer = "final_layer/dropout/Placeholder:0";
  Status run_status = session->Run({{input_layer, resized_tensor}},
                                   {output_layer}, {}, &outputs);`

The C++ code ends up with the following error message:
`Running model failed: Invalid argument: You must feed a value for placeholder tensor 'final_layer/dropout/Placeholder'`

What should I change in the above C++ code to remove this error?

**Comment by girving** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
